certbot-dns-cloudflare/certbot_dns_cloudflare/_internal/dns_cloudflare.py

```diff
@@ -151,9 +174,14 @@ def _find_zone_id(self, domain):
                 hint = None
 
                 if code == 6003:
-                    hint = 'Did you copy your entire API key?'
+                    hint = ('Did you copy your entire API token/key? To use Cloudflare tokens, '
+                            'you\'ll need python-cloudflare>=2.3.1. You have python-cloudflare {}'
+                            .format(CloudFlare.__version__))
                 elif code == 9103:
                     hint = 'Did you enter the correct email address?'
+                elif code == 9109:
+                    hint = ('Does your API token have "Zone:Zone:Read" and "Zone:DNS:Edit" '
+                            'permissions for all zones?')
```

Comment:
How do I set up my API token so I can hit this case to test it?

Reply:
Should've been somewhere else, fixed now :)